components/typeform/actions/list-responses/list-responses.mjs

```diff
@@ -157,17 +152,12 @@ export default {
       answered_fields: answeredFields?.join(","),
```

Comment:
Entering a comma separated list (e.g. `8dva5oqrctd9w31z8dva7ni347sqa35v,4jftg8jjvjcv2l7fcwo4jftg8la9o0d3`) in unstructured mode for the `type: "string[]"` props (like `includedResponseIds`) in this file gives an error like:
```
TypeError  includedResponseIds?.join is not a function
```

We could fix this error by using `.join` on these prop values only if they are arrays. For example:
```js
    included_response_ids: Array.isArray(includedResponseIds)
        ? includedResponseIds.join(",")
        : includedResponseIds,
```